nrc_ws/src/nrc_nav/src/nrc_drive_dr.py

```diff
@@ -97,28 +52,18 @@ def generate_drive_command(timer_event):
     start_time = time.time()
 
     # csv is generated with path, based on time passed since start
-    # TODO find exact filepath or make sure traj gen outputs the file to this directory. 
-    #       for now, I copied the file into this directory manually.
+    # will need to make sure to copy file into this directory after creating it in trajectory_gen
+    # will need to test how this works when the node is run from a launch file
```

Comment:
When running this code from not the src folder (i.e. a launch file, but I just used rosrun) the file cannot be found. The path to the ros package needs to be prepended to the file path. You can get the package's path with this: https://wiki.ros.org/Packages#Client_Library_Support